doc/index.md

```diff
@@ -94,7 +94,7 @@ below.
     </tr>
     <tr>
       <td valign="top"><code>@build_bazel_rules_apple//apple:apple.bzl</code></td>
-      <td valign="top"><code><a href="rules-general.md#apple_framework_import">apple_framework_import</a></code><br/></td>
+      <td valign="top"><code><a href="rules-general.md#apple_dynamic_framework_import">apple_dynamic_framework_import</a></code><br/></td>
```

Comment:
can you add an entry for `apple_static_framework_import` too?

Reply:
done